stories/exercises/05-customize-theme-tokens.mdx

```diff
@@ -0,0 +1,123 @@
+import { Meta, ColorPalette, ColorItem } from "@storybook/blocks";
+
+# 05. Customize with theme tokens
+
+With content on our landing page, we're going to make sweeping changes using USWDS Design Tokens and theme settings.
```

Comment:
**thought:** I've tried stayed away from possessive language like `our`, but no particular reason. Unsure about the pros/cons of either.

Reply:
I agree, resolved in https://github.com/Bixal/storybook-rapid-prototyping/pull/13/commits/8f254fbfa90718131c701de5ba33f0ac75c6fe8e